### ISSUE TITLE
Metadata is not of a supported version (29)

### ISSUE THREAD

**Issue by Symbai** [djkaty/Il2CppInspector#197]
![image](https://user-images.githubusercontent.com/14368203/142220763-2329898a-bcb6-47f8-bfdd-27a12de8d747.png)

[metadata.zip](https://github.com/djkaty/Il2CppInspector/files/7555450/metadata.zip)

**Comment by SamboyCoding** [comment]
Depending on what you want to do with the binary, my [Cpp2IL](https://github.com/SamboyCoding/Cpp2IL) project may help, and does support metadata v29

**Comment by Symbai** [comment]
I need to get rva addresses from methods and offsets from properties. So I can locate the code in memory, can your project do this?

**Comment by SamboyCoding** [comment]
yes, it will output that information as attributes on the types and members in the output dummy assemblies.

**Comment by eebssk1** [comment]
> yes, it will output that information as attributes on the types and members in the output dummy assemblies.

Will it add support to generate disassembler script just like the inspector does?

**Comment by SamboyCoding** [comment]
Do you mean like a ghidra or IDA script? They're on my list of things to add eventually but they're not a priority for the moment. In addition, I'm currently rewriting the application so I don't expect to be in a position where I *could* add that for at least a month.
